proto/Dockerfile

```diff
@@ -1,10 +1,10 @@
 # Start with a NodeJS base image that also contains yarn.
-FROM node:16.14.2-buster as nodejsbuilder
+FROM node:22.8.0-bookworm as nodejsbuilder
```

Comment:
@jamaljsr, let me know if you think this NodeJS base image version is ok, or if you'd like me to use `18.20-bookworm` instead :)!

Reply:
I tested this and it works for me, so this is fine.

I would suggest bumping the image version in [Dockerfile](https://github.com/lightninglabs/lightning-terminal/blob/master/Dockerfile) and [dev.Dockerfile](https://github.com/lightninglabs/lightning-terminal/blob/master/dev.Dockerfile) as well, just to be consistent.